trino/manifest.json

```
@@ -0,0 +1,35 @@
+{
+  "display_name": "Trino",
+  "maintainer": "@ndrluis",
+  "manifest_version": "1.0.0",
+  "name": "trino",
+  "metric_prefix": "trino.",
+  "metric_to_check": "trino.memory.reserved_distributed_bytes",
+  "creates_events": false,
+  "short_description": "Collects performance and usage stats on Trino clusters",
+  "guid": "f82a7dd0-5d8a-4d1c-ac99-5463da28d890",
+  "support": "contrib",
+  "supported_os": [
+    "linux",
+    "mac_os",
+    "windows"
+  ],
+  "public_title": "Trino Integration",
```

I'm sorry about this since this was part of our previous suggestion, but it was recently decided to remove the word "Integration" from all integration titles, cf. https://github.com/DataDog/integrations-extras/pull/1350.

```suggestion
  "public_title": "Trino",
```